jsettlers.mapcreator/src/jsettlers/mapcreator/localization/labels_ru.properties

```diff
@@ -12,100 +13,105 @@
-landscapedescr  = %s
-changeheightdescr   = изменить высоту
-increaseheightdescr = увеличить высоту
-decreaseheightdescr = уменьшить высоту
-flatten             = разравнить
-fixheights          = Исправить горы и ресурсы
-setstartpoint       = Установить стартовую позицию
-place_stacks        = %s
-place_resource      = %s
-remove_resource     = удалить ресурсы
-
-fuzzy_line          = размыто
-line                = линия
-circle_line         = круг
-grid_circle         = круг (меньше)
-grid_line           = линия (меньше)
-noisy_circle_line   = шум
-points              = точка
-line_inner          = внутренняя площадь (%)
-
-save = Сохранить
-undo = Назад
-redo = Вперёд
-errors          = Ошибки
-errors_n        = ошибок %d
-
-toolbox = Инструменты
-category-landscape = ландшафт
-category-heigths = горы
-category-land-resources = ресурсы
-category-objects = объекты
-category-settlers = поселенцы
-category-worker = рабочий
-category-specialist = специалист
-category-soldier = солдат
-category-materials = материалы
-category-mat-build = здание
-category-mat-food = пища
-category-mat-resources = ресурсы
-category-mat-tools = инструменты
-category-mat-weapons = оружие
-category-buildings = здания
-category-resources = ресурсы
-category-food = промысел
-category-military = военные
-category-social = общественные
-presets = предустановки
-preset-start = начальные объекты
-preset-wood = лесные работы
-
-landscape_GRASS      = трава
-landscape_DRY_GRASS      = сухая трава
-landscape_SAND      = песок / пляж
-landscape_FLATTENED      = плоский
-landscape_DESERT      = пустыня
-landscape_EARTH      = земля
-landscape_MOUNTAIN   = горы
-landscape_SNOW       = снег
-landscape_MOOR       = болото
-landscape_FLATTENED_DESERT       = плоская пустыня
-landscape_SHARP_FLATTENED_DESERT       = плоская пустыня (резкие края)
-landscape_GRAVEL       = гравий
-landscape_RIVER1 = река 1
-landscape_RIVER2 = река 2
-landscape_RIVER3 = река 3
-landscape_RIVER4 = река 4
-landscape_WATER1 = озеро/море 1
-landscape_WATER2 = озеро/море 2
-landscape_WATER3 = озеро/море 3
-landscape_WATER4 = озеро/море 4
-landscape_WATER5 = озеро/море 5
-landscape_WATER6 = озеро/море 6
-landscape_WATER7 = озеро/море 7
-landscape_WATER8 = озеро/море 8
-
-object_PLANT_DECORATION  = растения (декорация)
-object_DESERT_DECORATION = декорации в пустыне
-
-current-player = текущий игрок #
-statistics = статистика
-settings = настройки
-shapes = формы
-
-map-name = название карты
-map-description = описание карты
-max-width = ширина
-max-height = высота
-map-min-player = минимум игроков
-map-max-player = максимум игроков
+#
+# ** Tips to translate **
+# May use http://essiembre.github.io/eclipse-rbe/ as editor, makes it a lot easyer
+# Use a dot (.) to group tags by Category, this is common in property files, and then
+# it's displayed as groups in the Translator plugin mentioned above
+
+action.redo          = \u00D0\u0092\u00D0\u00BF\u00D0\u00B5\u00D1\u0080\u00D1\u0091\u00D0\u00B4
```

Comment:
We should have normal UTF-8 characters here instead of the \u... characters. You cannot read these texts. When the file is saved with UTF-8 (which should be the case anyways), we can simply use the correct UTF-8 letter.